pyproject.toml

```diff
@@ -1,7 +1,20 @@
-[tool.isort]
-profile = "black"
-multi_line_output = 3
-extend_skip_glob = ["*venv*", "server/api/proto"]
+[tool.ruff]
+select = [
+    "F", # pyflakes
+    "W", # pycodestyle
+    "E", # pycodestyle
+    "I", # isort
+]
+extend-exclude = [
```

Comment:
is `*venv*` not needed?

Reply:
No - this pattern is in `.gitignore`, which is respected by default.
https://docs.astral.sh/ruff/settings/#respect-gitignore
https://github.com/mlrun/mlrun/blob/f4b97867841685ff91fa3e8a162030b93f68dbc6/.gitignore#L29